### PR TITLE
Fixing typo and grammatical errors

### PR DESCRIPTION
**Pull request by sonaljain067**
Signed-off-by: Sonal Jain <sonal.jain6431@gmail.com>

**Description**

Fixing typo and grammatical errors

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ✔️ ] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,27 +60,27 @@ The following list of instructions pertain to commonplace site updates by contri
 
   1. It is recommended to use `@react-icons/all-files` instead of `react-icons` for importing icons. The issue with react-icons is that even though we are importing a single icon from the package/folder it still imports the complete icon folder which is creating unwanted junks of JS in the build.
   2. It is recommended to use Gatsby's `<StaticImage>` instead of `<img>` tag to display static images on the site. This performs automatic image optimization, thereby improving site performance.
-  3. Avoid creating duplicate copies of same image under different folders. All images must be imported from `/assets/images` folder. 
+  3. Avoid creating duplicate copies of the same image under different folders. All images must be imported from `/assets/images` folder. 
 
 ## Adding a Blog Post
 
-  1. In order to contribute a blog post, fork this repository, clone it, create a new branch and navigate to the `src/collections/blog` directory.
+  1. To contribute a blog post, fork this repository, clone it, create a new branch and navigate to the `src/collections/blog` directory.
```
Comment (Abhijay007) [inline CONTRIBUTING.md:67]: I think this change can be reverted

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -60,27 +60,27 @@ The following list of instructions pertain to commonplace site updates by contri
...
   2. Create a copy of the [blog template](https://github.com/layer5io/layer5/tree/master/src/collections/blog/blog-template).
   3. Follow the instructions included in the blog template and name the new file after the title of the blog article.
   4. Entries will be listed in chronological order automatically.
 
 ## Adding News
 
-  1. In order to add/update news items, fork this repository, clone it, create a new branch and navigate to the `src/collections/news` directory.
+  1. To add/update news items, fork this repository, clone it, create a new branch, and navigate to the `src/collections/news` directory.
```
Comment (Abhijay007) [inline CONTRIBUTING.md:74]: This one too, else LGTM

**Comment by sonaljain067** [comment]
Done, reverting both the changes!

**Comment by l5io** [comment]
🚀 Preview for commit e47180de8e3a37a0d3b652425e57537a22233f4a at: https://624afb77929ef41c51e085c0--layer5ng.netlify.app

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -71,16 +71,16 @@ The following list of instructions pertain to commonplace site updates by contri
...
   4. Entries will be listed in chronological order automatically.
 
 ## Updating/Creating a Community Member Profile 
 
-Layer5 community members are an integral part of what makes Layer5 and it's projects successful. Prominently highlighting our members and their works is something that we think is important. When adding a new or updating an existing community member profile, be sure to use the [Community Member Profile issue template](https://github.com/layer5io/layer5/issues/new?assignees=&labels=area%2Fcommunity&template=New+Member+Profile.md&title=%5BCommunity%5D+Member+Profile%3A) which has all the instructions that one may need for this.
+Layer5 community members are an integral part of what makes Layer5 and its projects successful. Prominently highlighting our members and their works is something that we think is important. When adding a new or updating an existing community member profile, be sure to use the [Community Member Profile issue template](https://github.com/layer5io/layer5/issues/new?assignees=&labels=area%2Fcommunity&template=New+Member+Profile.md&title=%5BCommunity%5D+Member+Profile%3A) which has all the instructions that one may need for this.
```
Comment (leecalcote) [inline CONTRIBUTING.md:81]: Good eye, @sonaljain067 :thu

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:81]: @leecalcote good to go through?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ We are beyond excited to see that you want to contribute! We would love to accep
   - [Contributing to Layer5 Projects](#contributing-to-layer5-projects)
   - [Contributing to Layer5's Blogs](#contributing-to-layer5s-blogs)
   - [How to Contribute](#how-to-contribute)
-      - [Prequisites](#prequisites)
+      - [Prerequisites](#prequisites)
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:9]: You will have to fix the `#prequisites` as well.

Comment (sonaljain067) [reply CONTRIBUTING.md:9]: Yeah, would do that!

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**